x-pack/plugins/security_solution/cypress/tsconfig.json

```diff
@@ -1,10 +1,11 @@
 {
-  "extends": "../../../../tsconfig.json",
+  "extends": "../../../../tsconfig.base.json",
   "exclude": [],
   "include": [
     "./**/*"
   ],
   "compilerOptions": {
+    "tsBuildInfoFile": "../../../../build/tsbuildinfo/security_solution/cypress",
```

Comment:
Just to understand, why this wasn't added in some of the config files (such as `packages/kbn-monaco/tsconfig.json`)?

Reply:
`tsc` emits it in `outDir` by default. For cases when we don't emit code or emit via `declarationDir`, an explicit `tsBuildInfoFile` path is required. Otherwise, the cache file will be written in the same folder when `tsconfig.json` is.